monopod/pkg/commands/scaffold.go

```diff
@@ -69,15 +45,15 @@ func Scaffold() *cobra.Command {
   # Generate a test image with run-as, user-gid, and group-gid
   monopod scaffold --package-name test --entrypoint /usr/bin/test --run-as 65530 --user-gid 65534 --group-gid 65534`,
 
-		RunE: func(_ *cobra.Command, _ []string) error {
-			return o.runScaffold()
+		RunE: func(cmd *cobra.Command, args []string) error {
+			o.ModuleName = args[0]
+			return o.runScaffold(cmd.Context())
 		},
-		Args: cobra.NoArgs,
+		Args: cobra.ExactArgs(1),
```

Comment:
nit, but we should probably update the documentation above to specify that you must specify the module-name.